### PR TITLE
Converting FAQ to English

### PR DESCRIPTION
**Pull request by srbala**
Referring to https://github.com/ant-design/ant-design-pro/issues/120, this is an attempt converting FAQ page to english. 

@xueqingxiao

### CODE REVIEW

**Comment by srbala** [comment]
Used google transulate for doc update, please make correction if necessary.

File: docs/faq.en-US.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Open the `sass` configuration in `.webpackrc`, see [sass] (https://github.com/sorrycc/roadhog#sass).
+
+### Git commit error?
+
+<img src="https://gw.alipayobjects.com/zos/rmsportal/KkPUhMMpGtEdhSGfxxKz.png" width="600" />
+
+Scaffolding defaults to the [eslint](http://eslint.org/) code style check. Please follow the prompts and resubmit it, or you can manually check `npm run lint`.
+
+### How to disable browser automatically when npm start is disabled?
```
Comment (afc163) [inline docs/faq.en-US.md]: disable browser opening automatically

File: docs/faq.en-US.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Modify `scripts.start` in `package.json` to:
+
+```js
+"start": "cross-env BROWSER=none roadhog server",
+```
+
+### Does the site support internationalization?
```
Comment (afc163) [inline docs/faq.en-US.md]: Does Ant Design Pro support internationalization?

File: docs/faq.en-US.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+This is one of the features of Ant Design Pro. The first version is currently available in Chinese. Internationalization will be our top priority in the next step.
```
Comment (afc163) [inline docs/faq.en-US.md]: one of major features

Comment (afc163) [reply docs/faq.en-US.md]: `will be our top priority in the next step.`

=>

`is in our 2.0 plan and will be released soon.`

**Pull request merged**